### ISSUE TITLE
Mouse/touch in-between particles in a bucket makes drag pick-up feel broken

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/build-an-atom#17]
When trying to grab a particle from a bucket, there are the empty spaces between three spheres that don't respond. It's technically correct, but it makes it seem like you "grabbed" a particle and it didn't drag (espsecially on touch platforms).

See the areas here:
![baa-inbetween](https://f.cloud.github.com/assets/480878/875973/288affd4-f8b4-11e2-8b01-35c6bb463381.png)

I'm not sure of the best way of handling this, but I think it would be nice from a usability point.

**Comment by Nancy-Salpepi** [comment]
I didn't run into any issues when playing with the sim on the iPad. On main, I zoomed in and still never came across a time where I went to grab a particle and came up empty handed (or fingered 🙂).

On my mac, the pointer changes from a grab hand to an arrow when it is over an empty space area. I see this same behavior in Build a Nucleus, Build a Molecule, and Isotopes and Atomic Mass. 

<img width="426" alt="Image" src="https://github.com/user-attachments/assets/74d06ce7-9160-4ed2-ba2e-6b1187389c66" />

I don't think anything needs to be here.

**Comment by arouinfar** [comment]
> I don't think anything needs to be here. Do we agree this can be closed?

I agree @Nancy-Salpepi. If there were usability issues, it would have been reported by now, as the HTML5 version of BAA has been available for more than a decade. Closing.
